srcpkgs/mutter/template

```
@@ -49,7 +49,7 @@ mutter-devel_package() {
 	pkg_install() {
 		vmove usr/include
 		vmove usr/lib/pkgconfig
-		vmove "usr/lib/mutter-7/*.gir"
+		vmove "usr/lib/mutter-8/*.gir"
```

you could glob as well

---

I better keep this as it is. I this could result in silent errors.

---

glob has a bug, those files will be moved into `${DESTDIR}/usr/lib/mutter-*/` literally (I've run into this myself).

---

agree @Gottox